pkg/blob/fetch.go

```diff
@@ -57,6 +68,27 @@ func (f *Fetcher) Fetch(dir string, blobURL string, stripComponents int) error {
 		return err
 	}
 
+	projectMetadataFile, err := os.Create(path.Join(metadataDir, "project-metadata.toml"))
```

Comment:
Should `"project-metadata.toml"` be defined as a constant somehwere?  We use that string in this function and in the associated tests.

Reply:
💯. But I would like to address this in a follow-up PR. I was going to put this const in the `github.com/buildpacks-community/kpack/pkg/cnb` package, but ran into #1473 and that requires some decent refactoring that I would rather not be in this PR.

Reply:
Makes sense to me